packages/apps/examples/nft-sdk/.env.example

```diff
@@ -0,0 +1,4 @@
+NX_GRAPHQL_API_URI=http://localhost:4000/graphql
```

Comment:
I wouldn't create `.env` files to manage a library. IMO, all this should be controlled by input parameters. 
The difference is that by parameters it will allows users to have more than 1 SDK instance at the same time

Reply:
This is an example app, I renamed this to `sdk-api` based on our offline discussion and I added some docs (in the libs/sdk README) on how to use this example app, I think this is the best way we have to develop right now